tests-e2e/cypress/e2e/edx-frontend-district/districtContacts.cy.ts

```diff
@@ -16,6 +16,19 @@ describe('District Contacts Page', () => {
   context('As an EDX district admin', () => {
     beforeEach(() => cy.login());
 
+    it('creates new district contact', () => {
+      cy.visit('/');
```

Comment:
Here we are visiting "/" and immediately manipulating the interface.  This may cause intermittent timeouts.  Have you experienced any?

Reply:
Great point, tested code again around 4 times; seems to be working fine so far